app/views/pages/home.html.haml

```diff
@@ -67,6 +67,36 @@
         .small-12.medium-6.columns
           = link_to 'Start a Hack Club', apply_path, class: 'button outline-outward'
 #home-map-jumbotron
+.main-section.sponsors
+  %h1 Sponsors
+  %ul.small-block-grid-2.medium-block-grid-3.logos
+    %li
+      = link_to "https://taplytics.com/" do
+        %img{alt: 'Taplytics', src: '/assets/taplytics.svg'}
+    %li
+      = link_to "http://www.hackreactor.com/" do
+        %img{alt: 'Hack Reactor', src: '/assets/hack_reactor.png'}
+    %li
+      = link_to "https://workflowy.com/" do
+        %img{alt: 'Workflowy', src: '/assets/workflowy.png'}
+    %li
+      = link_to "https://sourcegraph.com/" do
+        %img{alt: 'Sourcegraph', src: '/assets/sourcegraph.png'}
+    %li
+      = link_to "https://www.makeschool.com" do
+        %img{alt: 'Make School', src: '/assets/make_school.svg'}
+    %li
+      = link_to "https://www.nitrous.io/" do
+        %img{alt: 'nitrous.io', src: '/assets/nitrousio.svg'}
+    %li
+      = link_to "http://www.rackspace.com/" do
+        %img{alt: 'Rackspace', src: '/assets/rackspace.svg'}
+    %li
+      = link_to "https://codehs.com/" do
+        %img{alt: 'CodeHS', src: '/assets/codehs.png'}
+    %li.last
+      = link_to "http://firstcodeacademy.com/" do
+        %img{alt: 'First Code Academy', src: '/assets/first_code_academy.png'}
```

Comment:
Will using the raw `img` tag work with the asset pipeline in production? We can use the `image_tag` (http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag) which will link to the correct, renamed asset in production.